node/runtime/src/lib.rs

```diff
@@ -554,7 +563,7 @@ impl Runtime {
                 sender_id,
                 receiver_id,
             );
-            if let Ok(_) = result {
+            if result.is_ok() {
                 receiver.amount = wasm_res.balance;
```

Comment:
This one is the amount without stake. You should add stake back

Reply:
Fixed in 0354228